### PR TITLE
Pipeline options

### PR DESCRIPTION
**Pull request by ptzafrir**
Provide two configuration options for pipeline users when using acceptGitlabMergeRequest step

- useMRDescription, which will build   a commit message similar to gitlab "include description in merge commit" option.
- removeSourceBranch to delete source branch after accepting the Mr

### CODE REVIEW

**Comment by omehegan** [comment]
Thanks for the PR, it sounds good! I'll try to get it tested and into the next release.

File: src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java [hunk 1]
```diff
@@ -89,7 +105,19 @@ protected Void run() throws Exception {
             return null;
         }
 
-        private void println(String message) {
+        private String getCommitMessage(MergeRequest mergeRequest) {
+        	if (!step.useMRDescription)
+        		return step.mergeCommitMessage;
+        	
+        	println("Merge description is: "+mergeRequest.getDescription());
```
Comment (mattiasb) [inline src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java]: Should this be using the `LOGGER` object instead?

Comment (ptzafrir) [reply src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java]: This is calling the println method defined below, which will use LOGGER, or send output to listener if there is a task listener in use.

Comment (ptzafrir) [reply src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java]: Looking at it again, it seems like a debug comment that I wouldn't like to include (will output needless data to the job console). I removed it. Thanks for catching this.

Comment (mattiasb) [reply src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java]: Oh ok! :) Didn't see that one.

**Comment by omehegan** [comment]
@mattiasb thanks a lot for reviewing this!

**Comment by mattiasb** [comment]
@omehegan Oh that was just a drive-by comment. GitHub makes this up to be more than it was. :)

**Comment by omehegan** [comment]
@mattiasb it's more review than I did :)

**Comment by omehegan** [comment]
@ptzafrir I would like to get this merged for release soon, but I have two requests. First, can you add a note in the README which explains the new options? Second, I see you are adding a "remove source branch" option. We had an open feature request for this, which was closed recently because GitLab has added this feature on their side. See #181. Since your feature for that would only apply to Pipeline jobs, maybe we should just leave it out?

**Comment by ptzafrir** [comment]
I've updated the README.md. As for the option to remove source branch, in my use case I work with a team with a strict policy to remove all feature branches following merge. Some team members forget to manually check the checkbox when creating MR and so we are left to manually cleanup every once in a while.

**Comment by omehegan** [comment]
@ptzafrir thanks! I will merge this now, but I just released a few other things earlier today so I will hold this for the next release. Should be in a couple of weeks.

**Comment by omehegan** [comment]
Released as 1.5.8 today because we had an unrelated breaking change in 1.5.7.

**Pull request merged**